etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -32,16 +32,22 @@ id=mh_default_org
 #
 name=Opencast Project
 
-# Server name that identifies the organization (tenant) on this Opencast installation.
+# Server names that identifies the organization (tenant)
```

Comment:
I'm also not sure whether "identify" is really the best choice of word here. The organization can have more hosts than are listed here, as you correctly state below, namely those potentially listed by other nodes.

Maybe something like:

> Server names that **belong** to the organization (tenant)